Runtime/Interaction/CustomMeshUtils/HPUICustomMesh.cs

```diff
@@ -0,0 +1,26 @@
+using ubco.ovilab.HPUI.Interaction;
+using UnityEngine;
+
+namespace ubco.ovilab.HPUI.CustomMeshUtils
+{
+    [RequireComponent(typeof(CustomMeshCollidersManager))]
+    public class HPUICustomMesh : HPUIContinuousInteractable
```

Comment:
We may want to consider the naming. Consider `HPUIStaticContinuousInteractable`, with the remaining classes/methods reflecting that. This is not a Mesh, it's an Interactable!

More importantly, this is not overriding `ComputeInteractorPostion`, which will break this when interactions happen.

You may want to also consider inheriting from `HPUIBaseInteractable` instead of `HPUIConitnuousInteractable`, you are practically overriding everything in `HPUIContinouousInteractable`.